### DISCUSSION TITLE
Is it guaranteed that the code styles will not evolve ?

### DISCUSSION THREAD

**Discussion by IonasNeonakis** [facebookincubator/ktfmt#415] [Q&A] [answered]
Currently I'm using ktlint as a code formatter but the biggest problem is that I cannot fix the version of the intelliJ plugin. So my plugin's rules are not the same as the one in my CI since I have two different versions. 

I'm looking to migrate to an other formatter and ktfmt caught my attention. 

I'd like to use a 4 ident style format so I will be using DROPBOX_STYLE. 

However I'd like to know what will happen if that style will evolve ? Is it a fixed code style and won't be changed in the future ? Or maybe one day the style will change to 2 indent per say and I'll have to reformat my entire codebase again ?

**Comment by JavierSegoviaCordoba** [comment, marked as answer]
Code style evolves but there are different ones.

For example the official Kotlin style uses 4 spaces and only if the official specs change to 2 spaces ktfmt would change.

**Reply by hick209** [reply]
Exactly this 👆

I believe it's quite unlikely that the indent space would ever change, but other factors, like trailing commas being default as it's being discussed in https://github.com/facebook/ktfmt/issues/442, could happen.
